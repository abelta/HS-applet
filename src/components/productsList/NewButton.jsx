import React from 'react';
import PropTypes from 'prop-types';

const NewButton = ({ onClick }) => <button onClick={onClick}>New</button>;

NewButton.propTypes = {
  onClick: PropTypes.func.isRequired,
};

export default NewButton;
