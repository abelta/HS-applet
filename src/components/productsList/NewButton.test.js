import React from 'react';
import renderer from 'react-test-renderer';
import NewButton from './NewButton';

describe('NewButton', () => {
  const tree = renderer.create(<NewButton onClick={jest.fn} />).toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});
