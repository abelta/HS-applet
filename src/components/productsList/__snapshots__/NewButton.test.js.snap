// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewButton renders correctly 1`] = `
<button
  className="waves-effect waves-light btn-flat blue white-text"
  onClick={[Function]}
>
  New
</button>
`;
